File: cpp/operaciones.cpp
#include <iostream>

using namespace std;

int main() {
    int A, B;
    
    // Solicitar los valores al usuario
    cout << "Ingrese valor A: ";
    cin >> A;

    cout << "Ingrese valor B: ";
    cin >> B;

    // Realizar operaciones
    int suma = A + B;
    cout << "La suma de los numeros es: " << suma << endl;

    int res = A - B;
    cout << "La resta de los numeros es: " << res << endl;

    int multi = A * B;
    cout << "La multiplicación de los numeros es: " << multi << endl;

    float div = static_cast<float>(A) / B;  // Para obtener un resultado con decimales
    cout << "La división de los numeros es: " << div << endl;

    bool igual = (A == B);
    cout << "¿Los numeros son iguales? " << (igual ? "Sí" : "No") << endl;

    bool mayor = (A > B);
    cout << "El número menor es: " << (mayor ? B : A) << endl;
    cout << "El número mayor es: " << (mayor ? A : B) << endl;

    return 0;
}

File: JavaScript/operaciones.js
const readline = require('readline');

// Crear la interfaz de lectura de consola
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

// Solicitar los valores al usuario
rl.question('Ingrese valor: ', (A) => {
    rl.question('Ingrese valor: ', (B) => {
        // Convertir los valores a enteros
        A = parseInt(A);
        B = parseInt(B);

        // Realizar las operaciones
        let suma = A + B;
        console.log('La suma de los numeros es: ' + suma);

        let res = A - B;
        console.log('La resta de los numeros es: ' + res);

        let multi = A * B;
        console.log('La multiplicación de los numeros es: ' + multi);

        if (B !== 0) {
            let div = A / B;
            console.log('La división de los numeros es: ' + div);
        } else {
            console.log('No se puede dividir entre cero');
        }

        let igual = (A === B);
        console.log('El número ' + (igual ? 'es' : 'no es') + ' igual: ' + A);

        if (A > B) {
            console.log('El número menor es: ' + B);
            console.log('El número mayor es: ' + A);
        } else if (A < B) {
            console.log('El número menor es: ' + A);
            console.log('El número mayor es: ' + B);
        } else {
            console.log('Los dos números son iguales');
        }

        // Cerrar la interfaz de lectura
        rl.close();
    });
});
